integration-tests/gradle/build.gradle

```diff
@@ -64,6 +64,11 @@ test {
     }
     // Native Build test are run in an other task
     exclude 'io/quarkus/gradle/nativeimage/*.class'
+
+    reports {
+        // https://github.com/ScaCap/action-surefire-report#tips-for-gradle
+        junitXml.enabled = true
```

Comment:
@glefloch FYI

Reply:
This is not the case by default? 
If not, I think you should also do it in the `nativeTest` block which is also a test tasks (line 78).

Reply:
Ok, I'll have a look.

Reply:
You're right! I removed this change. Thanks for the pointer!